CMakeLists.txt

```diff
@@ -117,6 +121,8 @@ include_directories(
   "./generated"
   "./imports/include"
   "./source"
+  "./third_party/grpc-sideband/src"
+  "./third_party/grpc-sideband/moniker_src"
```

Comment:
If grpc-sideband was updated to using "Modern" CMake with targets. It could specify its header files locations so that components that depend on the library wouldn't need to add these themselves. See [target_include_directories](https://cmake.org/cmake/help/latest/command/target_include_directories.html)

Something like the following in [grpc-sideband/CMakeLists.txt](https://github.com/ni/grpc-sideband/blob/b1cd688921128044054592315327b7768e7f894c/CMakeLists.txt#L35)
```cmake
target_include_directories(ni_grpc_sideband
    PUBLIC
	src
        moniker_src)
```

Reply:
I've a PR out in the sideband repo for this https://github.com/ni/grpc-sideband/pull/9 which is waiting for approval.
Can we submit this change as is and we would clean it up as a follow-up when the other PR goes in?
This would help us get unblocked on other work items.